lib/jormungandr/test/unit/Cardano/Wallet/Jormungandr/ApiSpec.hs

```diff
@@ -111,3 +197,20 @@ spec = do
             return ()
   where
     decodeJSON = eitherDecode :: BL.ByteString -> Either String StakeApiResponse
+
+{-------------------------------------------------------------------------------
+                                  Test data
+-------------------------------------------------------------------------------}
+
+testPoolId1 :: PoolId
+testPoolId1 = mkTestPoolId
+    "c780f14f9782770014d8bcd514b1bc664653d15f73a7158254730c6e1aa9f356"
+
+testPoolId2 :: PoolId
+testPoolId2 = mkTestPoolId
+    "653f9aa1e6c0374528517a37f51d356466cb1b415dcb8d4100772879f41f087c"
+
+mkTestPoolId :: Text -> PoolId
+mkTestPoolId = either (error textError) id . fromText
```

Comment:
What about calling this `unsafePoolId`, and moving it with other unsafe constructors in `Cardano.Wallet.Unsafe` for better re-usability and consistency ?

Reply:
@KtorZ wrote:
> What about calling this `unsafePoolId`, and moving it with other unsafe constructors in `Cardano.Wallet.Unsafe` for better re-usability and consistency ?

Done: https://github.com/input-output-hk/cardano-wallet/pull/995/commits/1e477484deeaa043605a137ed7b3f9429cc0f472